docs/howto/event-prep.md

```diff
@@ -0,0 +1,133 @@
+```
+
+During events, we want to tilt the balance towards reducing server startup time. The docs at [](topic:resource-allocation) have more information about all the factors that should be considered during resource allocation.
+
+Assuming this hub already has a profile list, before an event, you should check the following:
+
+1. **Information is available**
+
+    Make sure the information in the event GitHub issue was filled in, especially the number of expected users before an event and their expected resource needs (if that can be known by the community beforehand).
+
+2. **Given the current setup, calculate how many users will fit on a node?**
+
+    Check that the current number of users/node respects the following general event wishlist.
+
+3. **Minimize startup time**
+
+  - have at least `3-4 people on a node` as few users per node cause longer startup times, but [no more than ~100]( https://kubernetes.io/docs/setup/best-practices/cluster-large/#:~:text=No%20more%20than%20110%20pods,more%20than%20300%2C000%20total%20containers)
+  - don't have more than 30% of the users waiting for a node to come up
+
+    ````{admonition} Action to take
+    :class: tip
+
+    If the current number of users per node doesn't respect the rules above, you should adjust the instance type so that it does.
+    Note that if you are changing the instance type, you should also consider re-writing the allocation options, especially if you are going with a smaller machine than the original one.
+
+    ```bash
+    deployer generate resource-allocation choices <instance type>
+    ```
+    ````
+
+4. **Don't oversubscribe resources**
+    The oversubscription factor is how much larger a limit is than the actual request (aka, the minimum guaranteed amount of a resource that is reserved for a container). When this factor is greater, then a more efficient node packing can be achieved because usually most users don't use resources up to their limit, and more users can fit on a node.
+
+    However, a bigger oversubscription factor also means that the users that use more resources than they are guaranteed can get their kernels killed or CPU throttled at some other times, based on what other users are doing. This inconsistent behavior is confusing to end users and the hub, so we should try and avoid this during events.
+
+    ````{admonition} Action to take
+    :class: tip
+
+    For an event, you should consider an oversubscription factor of 1. For this you can use the deployer script by passing it the instance type where the pods will be scheduled on (in this example is `n2-highmem-4`), then from its output, pick the choice(s) that will be used during the event based on expected usage.
+
+    ```bash
+    deployer generate resource-allocation choices n2-highmem-4
+    ```
+    ````
+
+
+#### 3.2. Setting a minimum node count on a specific node pool
+TODO
```

Comment:
````suggestion
```{warning}
This section is a Work in Progress!
```
````

I'd maybe make this a callout so when it's rendered it's a bit clearer? Same below.